intel_pytorch_extension_py/ops/dil_linear_relu.py

```diff
@@ -0,0 +1,55 @@
+import torch
+from torch import nn
+from torch.nn.parameter import Parameter
+from torch.nn import init
+from torch.autograd import Function
+import math
+import _torch_ipex as core
+
+class dilLinearFuseReluFC(Function):

```

Comment:
you can use LinearFuseRelu directly, not expose dil to front-end.

Reply:
Done